engine/src/main/java/org/terasology/rendering/logic/LightComponent.java

```diff
@@ -15,13 +15,14 @@
  */
 package org.terasology.rendering.logic;
 
-import org.terasology.reflection.metadata.FieldMetadata;
 import org.terasology.entitySystem.Component;
 import org.terasology.math.geom.Vector3f;
 import org.terasology.network.Replicate;
 import org.terasology.network.ReplicationCheck;
+import org.terasology.reflection.metadata.FieldMetadata;
 
 /**
+ * Add this component to an entity for it to transmit light from its location.  By default the component is configured to act similarly to a placed torch block.
  */
 // TODO: Split into multiple components? Point, Directional?
 public final class LightComponent implements Component, ReplicationCheck {
```

Comment:
Probably a good discussion we should all have is how "shiny" we want the world to be.  We do not have the option to set shinyness on a block level,  only at the light source.  The default value of 0.1 seems a little much for torch light when you drop it close to a wall.